controllers/drplacementcontrol_controller.go

```diff
@@ -926,20 +926,28 @@ func (r *DRPlacementControlReconciler) getDRPolicy(ctx context.Context,
 func (r DRPlacementControlReconciler) addLabelsAndFinalizers(ctx context.Context,
```

Comment:
(nit) The name is no longer tells the whole story. Not only we are adding label and finalizer but also annotation. Maybe we should rename it to something like; `updateMetadata` or `updateObjectMetadata` or something...

Reply:
Updated, was struggling for a name so used the above and added a brief comment.